### DISCUSSION TITLE
Scrum Prep Milestone 3-3

### DISCUSSION THREAD

**Discussion by NatiPerez459** [uprm-inso4101-2024-2025-s1/semester-project-mercado-colegial-online#205] [General]
Each team member will comment progress made through the past week.

**Comment by ActuallySeb** [comment]
Hellest of Hell Weeks

**Comment by Cabinetman** [comment]
Completed the Order model backend connection.

**Comment by CactusByte** [comment]
Completed some UI modifications and had 2 exams this week

**Comment by JadStelar** [comment]
Tested the password manager "pass" locally and documented its setup and usage.

**Comment by CovasJJ** [comment]
This week I completed the task on working on revising section 2.1.5 of the Documentation.

**Comment by gcal77** [comment]
This week I worked on the revision of section 2.2.4

**Comment by cristian-barreras** [comment]
I worked and finished the task of adding a profile page.

**Comment by JuanIranzo** [comment]
Implemented feedback on Section 2.1.4 of the documentation

**Comment by RyanVelez101** [comment]
Implemented updated welcome page for the UI.

**Comment by Kaariinaa08** [comment]
This week I completed a Lecture Topic Task that focused on seeing the hypothetical errors we might come across when we launch our website and how we can overcome them. These hypotheticals where then documented in our Milestone 3 Documentation Doc.

- Karina Lopez Rodriguez

**Comment by elliot-gomez** [comment]
This week, I created a document compiling all of the professor’s feedback on the Milestone 2 documentation. I organized the feedback by the specific sections it applied to, making it clear for the team which feedback needs to be addressed in each part of the document.
- Elliot J. Gomez Coriano

**Comment by franrov4** [comment]
This week, I had to opportunity to implement unit tests for our models. Discussed and resolved questions about the integration of routes, database relationships, and testing workflows.

**Comment by Christian2101** [comment]
Been working on fixing the documentation with the feedback provided by the professor

**Comment by FabiolaZTorres** [comment]
Fixed a bug where if the Login page was refresh it would load into the wrong page and reviewed tasks.

- Fabiola Z. Torres Maldonado

**Comment by JuanDelPueblo** [comment]
This week I completed task to move the old sign-up/login logic to the new components, ensuring the connection to the database worked.

**Comment by DiegoVazquez1** [comment]
This week i worked on 2 tasks. one moving files to the vue format and another making the whole profile page wihtout the database conected. Made the buttons and made it somewhat pretty

**Comment by AlvinBadillo** [comment]
This week I worked with updating a section of the documentation, I also started to work on a database connection.

**Comment by PerezMarcelo** [comment]
Planning for future task.

**Comment by YampierFernandez** [comment]
Created task to further backend connectivity

**Comment by CdoxHype** [comment]
Planning the redesign of a task of web page.
